### ISSUE TITLE
i18n WG & IG rechartering 2021

### ISSUE THREAD

**Issue by r12a** [w3c/strategy#271]
New charter proposal, reviewers please take note.

# Charter Review

[WG Charter:](https://www.w3.org/International/i18n-activity/i18n-wg/other-charters/proposed-charter-2021-2023.html)

[IG Charter:](https://www.w3.org/International/i18n-activity/i18n-ig/other-charters/proposed-charter-2021-2023.html)

What kind of charter is this?  This charter is essentially the same as the previous charter, given that the work of the i18n WG is ongoing, and there are no new Recommendation track deliverables.

* Existing 
 - [x] Existing WG recharter 
 - [x] Existing IG recharter

Communities suggested for outreach: 
none

Known or potential areas of concern: 
none

Where would charter proponents like to see issues raised? 
this strategy funnel issue

Anything else we should think about as we review? 
no

**Comment by himorin** [comment]
(ok to be labelled as i18n review completed??? as self review?)

For ruby annotation spec link, I think there is another WG NOTE which was merged into previous W3C one
https://www.w3.org/TR/2014/NOTE-html-ruby-extensions-20140204/
which should be the suitable one?

**Comment by r12a** [comment]
@himorin that note was merged into the W3C HTML spec (and vestiges of it found their way into the WhatWG HTML spec). It was created and published by the HTML WG.  I'm not clear why that would be relevant to the i18n WG charter(?)

**Comment by michael-n-cooper** [comment]
APA has no comments on these charters, and feels comfortable with its ability to coordinate with these groups. Over to @brewerj to complete accessibility horizontal review.

**Comment by himorin** [comment]
>  I'm not clear why that would be relevant to the i18n WG charter(?)

I neither, but ruby annotation spec is listed in the list of deliverables: https://www.w3.org/International/i18n-activity/i18n-wg/other-charters/proposed-charter-2021-2023.html#deliverables

**Comment by himorin** [comment]
Thank you for comment, @r12a . I now understand backgrounds.
One additional point, but quite minor, 2.1 timeline at the last of section 2 might be 2.6?

**Comment by samuelweiler** [comment]
Here are diffs for both charters:
[For the WG](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FInternational%2Fgroups%2Fwg%2Fcharter.html&doc2=https%3A%2F%2Fwww.w3.org%2FInternational%2Fi18n-activity%2Fi18n-wg%2Fother-charters%2Fproposed-charter-2021-2023.html)
[for the IG](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2FInternational%2Fgroups%2Fig%2Fcharter.html&doc2=https%3A%2F%2Fwww.w3.org%2FInternational%2Fi18n-activity%2Fi18n-ig%2Fother-charters%2Fproposed-charter-2021-2023.html)

I see that this issue says to file comments here.  The proposed WG charter (but not the IG charter) directs comments into Github.  For now, I'm just filing here, as requested.  Let me know if you want Github issues.

For the WG:

- The history section needs a link to the current/2019 charter.  https://www.w3.org/International/groups/wg/charter.html
- Please expand the IRI acronym the first time it is used
- What is meant by "Maintain positive feedback for outreach materials."?  I'm not sure I object, but this is a weird criterion.
- **Since the WG is maintaining Recs, presumably it need the HR (and testing?) boilerplate.  I note that the participation section mentions test leads, suggesting that there is testing.  (It might instead be appropriate to hav e a line about NOT testing, given what these specs are...)**
- It seems odd that a WG (v. people who happen to be members of a WG) would attempt to publish conference papers under its own name (Section 4.2).

For the IG: 

- The history section needs a link to the current/2019 charter. https://www.w3.org/International/groups/ig/charter.html
- s/Working Group Notes/Interest Group Notes/
- s/ on a public repository/ in public repositories/
- Do we really need the redundant mention of lists being public, in section 7?

Marking this as needs-resolution for the WG only, for the HR comment in bold above.

**Comment by r12a** [comment]
WG

>     The history section needs a link to the current/2019 charter. https://www.w3.org/International/groups/wg/charter.html

Done.

>     Please expand the IRI acronym the first time it is used

It was in an abbr element, which provided the expansion. Nevertheless, i took that out and expanded the acronym.

>     What is meant by "Maintain positive feedback for outreach materials."? I'm not sure I object, but this is a weird criterion.

It means to continue to produce great materials that people compliment us on and use.

>     Since the WG is maintaining Recs, presumably it needs the HR (and testing?) boilerplate. I note that the participation section mentions test leads, suggesting that there is testing. (It might instead be appropriate to hav e a line about NOT testing, given what these specs are...)

To be clear, we very much hope that we will not need to work on those maintenance specs and we have no plans or expectations for it to be necessary, so the boilerplate will probably not be needed. Nevertheless, i added it.

The testing we refer to is related to the i18n test suite, which tests support for i18n features in browsers.  We have no plans to work on any specs that would require feature testing.

>     It seems odd that a WG (v. people who happen to be members of a WG) would attempt to publish conference papers under its own name (Section 4.2).

I removed the offending phrase.

>     Previous WG charters (e.g. https://www.w3.org/International/core/charter-2016.html) need notes saying they're obsolete and pointing at the next-newest charter

I'll need to do that for the previous charter later, and will try to remember to do it for others that may need it.

**Comment by r12a** [comment]
For the IG:

>     The history section needs a link to the current/2019 charter. https://www.w3.org/International/groups/ig/charter.html

Done.

>     s/Working Group Notes/Interest Group Notes/

clarified: "by the Internationalization Working Group as Working Group Notes"

>     s/ on a public repository/ in public repositories/

Done.

>     Do we really need the redundant mention of lists being public, in section 7?

Maybe.

**Comment by r12a** [comment]
@brewerj do you have any comments arising from your discussion last week, or are we able to move forward?

**Comment by plehegar** [comment]
@r12a , any objection to start the W3M review for those charters?

**Comment by r12a** [comment]
No objection. I heard no objections from Judy, and the i18n WG is ready and waiting.

**Comment by plehegar** [comment]
https://lists.w3.org/Archives/Member/w3c-ac-members/2021JulSep/0051.html
